### PR TITLE
Adding energy scaling to mixer unit model

### LINKED ISSUE #458: Energy and pressure scaling in Mixer
The Mixer unit model currently does not add scaling for the energy and momentum balances.

### PR DESCRIPTION
**Pull request by andrewlee94**
## Summary/Motivation:
The core Mixer model currently lacks code to perform constraint scaling on the energy balance.

## Changes proposed in this PR:
- Add code to do constraint scaling on the Mixer energy balance if it exists
- Add a basic test for Mixer scaling

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by eslickj — DISMISSED** [review]
This looks right to me, as long as the enthalpy flow terms have scaling factors.  I think you can have min_scaling_factor generate a warning for that if you like.

**Comment by andrewlee94** [comment]
@eslickj `min_scaling_factor` generates a warning by default, and also has a default value of 1, so we should be good (or at least not bad).

**Review by TimBartholomew — DISMISSED** [review]
LGTM

File: idaes/generic_models/unit_models/mixer.py [hunk 1]
```diff
@@ -1022,3 +1022,14 @@ def calculate_scaling_factors(self):
                             _s = iscale.get_scaling_factor(ft, default=1)
                             s = _s if _s < s else s
                     iscale.constraint_scaling_transform(c, s, overwrite=False)
+
+        if hasattr(self, "enthalpy_mixing_equations"):
+            for (t), c in self.enthalpy_mixing_equations.items():
```
Comment (adam-a-a) [inline idaes/generic_models/unit_models/mixer.py:1027]:
```suggestion
            for t, c in self.enthalpy_mixing_equations.items():
```

File: idaes/generic_models/unit_models/tests/test_mixer.py [hunk 2]
```diff
@@ -1122,6 +1122,10 @@ def test_dof(self, sapon):
     def test_initialize(self, sapon):
         initialization_tester(sapon)
 
+    @pytest.mark.component
+    def test_saling(self, sapon):
```
Comment (adam-a-a) [inline idaes/generic_models/unit_models/tests/test_mixer.py:1126]:
```suggestion
    def test_scaling(self, sapon):
```

**Review by adam-a-a — APPROVED** [review]

**Review by TimBartholomew — APPROVED** [review]

**Review by eslickj — APPROVED** [review]

**Pull request merged**